src/compile/compile.ts

```diff
@@ -68,10 +92,13 @@ export function compileRootGroup(model: Model) {
     {
       from: {data: LAYOUT},
       properties: {
-        update: {
-          width: {field: 'width'},
-          height: {field: 'height'}
-        }
+        update: extend(
+          model.title() !== undefined ? {y: {'value': 55} }: {},
```

Comment:
55 actually looked way to large for the example @kanitw showed today. Can you try different examples and see whether we need a more complex formula here?